app/middlewares/validateToken.js

```diff
@@ -5,7 +5,8 @@ const { authenticationError } = require('../errors');
 exports.validateToken = (req, res, next) => {
   const { authorization } = req.headers;
   try {
-    decodeToken(authorization);
+    const token = decodeToken(authorization);
+    req.locals = { ...req.locals, ...token };
```

Comment:
You need token data in locals ? For what ? Spread is tricky too, if you need it let token as key